src/CoreBundle/Bridge/Chameleon/Module/StaticView/StaticViewModule.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Chameleon System (https://www.chameleonsystem.com).
+ *
+ * (c) ESONO AG (https://www.esono.de)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ChameleonSystem\CoreBundle\Bridge\Chameleon\Module\StaticView;
+
+use ChameleonSystem\CoreBundle\Service\LanguageServiceInterface;
+
+/**
+ * Can be used to display static information in the backend. The module requires the additional module config parameter
+ * "targetView" containing the path to the view to be displayed (relative to snippets-cms). This path name may contain a
+ * "[{language}]" placeholder. If present, this placeholder will be replaced with the current display language.
+ */
+class StaticViewModule extends \MTPkgViewRendererAbstractModuleMapper
+{
+    /**
+     * @var LanguageServiceInterface
+     */
+    private $languageService;
+
+    public function __construct(LanguageServiceInterface $languageService)
+    {
+        parent::__construct();
+        $this->languageService = $languageService;
+    }
+
+    public function Accept(\IMapperVisitorRestricted $visitor, $cachingEnabled, \IMapperCacheTriggerRestricted $cacheTriggerManager)
+    {
+        $targetView = $this->aModuleConfig['targetView'];
+        $targetLanguage = $this->languageService->getActiveLanguage()->fieldIso6391;
+        $targetView = \str_replace('[{language}]', $targetLanguage, $targetView);
```

Comment:
That is necessary due to legacy reasons?

Or is this a current technique for the backend?

Reply:
Do you mean the bracket notation? It is the established syntax for replacing placeholders, so I wouldn't introduce a change here (the only other candidate is Twig's `%...%` notation, but this isn't a template).

Reply:
No: that there might be multiple file version according to language.

And not for example a translation mechanism that prints the relevant translations - like with frontend twig (trans filter).

Reply:
There isn't any mechanism like this yet. The welcome texts will be complete views and therefore have multiple lines and also contain HTML, so this was really cumbersome if we tried to handle it via translation.